src/shared/devServerUtils.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import path from 'node:path';
+import { CommonUtils } from '@salesforce/lwc-dev-mobile-core';
+
+export class DevServerUtils {
```

Comment:
@maliroteh-sf @khawkins We don't need this class anymore if we are completely switching to leverage `sf config` to stash identity token and any other properties.

Reply:
Yes we should remove these if not needed anymore

Reply:
Removed it.